index.js

```diff
@@ -27,7 +29,12 @@ function camelCase(sentence) {
     arr[i] = arr[i].toLowerCase()
 
     const letters = arr[i].split('')
-    letters[0] = letters[0].toUpperCase()
+
+
+    if (i > 0) {
+      letters[0] = letters[0].toUpperCase()
```

Comment:
I'd put the `const letters = arr[i].split('')` inside the `if` too since you only need to do it if `i` is great than 0.